### ISSUE TITLE
Android Components Sticker Research

### ISSUE THREAD

**Issue by jonalmeida** [mozilla-mobile/android-components#1231]
We want to make a sticker for a-c that shows describes what we do in a single simple visual design.

## Ideas
1. Use [Material Design launcher icon][3] guidelines.
2. Abstract design - something with a small colour palette.
3. Include the Android logo in there some.

## Investigation
- [Searching for "components"][0] shows puzzle pieces or [circuit components][1].
- [Searching for "modular"][2] shows interconnected boxes.
- Solicit ideas from people with a better sense of design (cc: @brampitoyo @aminalhazwani 👋😅)

[0]: https://duckduckgo.com/?q=components+icon&iar=images&iax=images&ia=images
[1]: https://www.iconfinder.com/icons/381906/circuit_component_current_electronic_microcontrolller_power_technology_icon
[2]: https://duckduckgo.com/?q=modular+icon&iar=images&iax=images&ia=images
[3]: https://material.io/design/iconography/#

**Comment by jonalmeida** [comment]
Setting myself as assignee to make sure we get one made, but help/ideas are always appreciated!

**Comment by pocmo** [comment]
> frankenstein's monster (a bunch of things sewed together... IT'S ALIVE!)

@antlam once created this Zombie sticker :)

![img_20181030_182721](https://user-images.githubusercontent.com/89638/47737733-d6535200-dc71-11e8-87e2-191b144aa35e.jpg)

**Comment by jonalmeida** [comment]
> Oh and: Quantum had this Firefox Quantum Suit. Components are the pieces of the suit? (I used that for some slides in the past).

![](https://techspot-static-xjzaqowzxaoif5.stackpathdns.com/images2/news/bigimage/2017/08/2017-08-06-image-4.jpg)

[Higher res image](https://cnet2.cbsistatic.com/img/QppER9qu05sChuk5sJbg4g3G9iw=/1600x900/2017/11/14/561fdc54-32db-4d97-878f-e07bed63ba42/mozilla-mark-57-firefox-mecha-suit-from-mozilla-crop.jpg)

I like this idea too! Replace tiny foxes with Androids?

**Comment by pocmo** [comment]
Some cheesy things from my slides:

![screenshot_2018-10-30 suits and other things](https://user-images.githubusercontent.com/89638/47738351-40202b80-dc73-11e8-9af7-329b26938325.png)

![screenshot_2018-10-30 suits and other things 1](https://user-images.githubusercontent.com/89638/47738355-431b1c00-dc73-11e8-81cc-d1625e4a4e2d.png)

**Comment by pocmo** [comment]
Layer cake:
https://www.google.com/search?q=layer+cake&tbm=isch

**Comment by jonalmeida** [comment]
@gabrielluong had a nice idea: use a burst view of different components (quantum suit limbs) layering on top. A similar concept to the [3D view][1] we had in desktop.

@dnarcese said we should try making [transparent stickers][0] of individual pieces so that you could layer them on top of each other.

[0]: https://is.alicdn.com/kf/HTB1XZk9FFXXXXbYXFXXq6xXFXXXX/200991871/HTB1XZk9FFXXXXbYXFXXq6xXFXXXX.jpg
[1]: https://i.stack.imgur.com/dvdUS.png

**Comment by pocmo** [comment]
> @dnarcese said we should try making transparent stickers of individual pieces so that you could layer them on top of each other.

I like that. How about taking the UI of a browser an separating it into pieces (toolbar, engine view, awesomebar) and you can build your own browser with the stickers. That's the closest you can get to what the components are :D

**Comment by jonalmeida** [comment]
Alternative to layers, it could also be pieces like these British coins: https://www.reddit.com/r/DesignPorn/comments/92p8i1/all_the_british_coins_put_together_make_a_crest/

**Comment by jrconlin** [comment]
Hi all! From what I've been able to tell, various sticker sites have very different turn around speeds, so that might impact what designs you do. I'm pretty sure that more complicated sheets might also mean longer processing time, like doing a "kiss cut" for the multi-sticker sheets, or giving a sticker an unusual size. It might be good to have a couple of alternates that you can pick in case you have to adjust for schedule.

My order with StickerMule (100 3"x2" rectangle) took a very impressive 4 days. Ordered on Friday, shipped from Syracuse and supposed to be delivered today. I'll post images of the quality if you all like. I've also used StickerRobot in the past. They take a good deal longer, but I've been Super Impressed by their work. (The "Danger" stickers I have are from them). If you use StickerRobot, it might be cutting things a bit too close to make it for MozLando.

**Comment by jonalmeida** [comment]
> Hi all! From what I've been able to tell, various sticker sites have very different turn around speeds, so that might impact what designs you do. I'm pretty sure that more complicated sheets might also mean longer processing time, like doing a "kiss cut" for the multi-sticker sheets, or giving a sticker an unusual size. It might be good to have a couple of alternates that you can pick in case you have to adjust for schedule.

That's a good point. We should finalize these designs asap.

**Comment by jrconlin** [comment]
(The "weight" is adjustable if possibly a tad aspirational.)
Likewise happy to replace "Use Hooks" with whatever pithier statement you'd like.

**Comment by jrconlin** [comment]
A slightly altered version using a number I got from Dr. Brown.

![ac_box2](https://user-images.githubusercontent.com/35755/48043189-42cec500-e13a-11e8-9b3c-a532c2f03864.png)

**Comment by aminalhazwani** [comment]
Hello everyone, sorry for the late reply! I would suggest to put as little text as possible, and play more with metaphors rather than a descriptive approach.

For inspiration you can have a look at this Instagram account curated by @bwinton https://www.instagram.com/mozsticker/?hl=en. 

Regarding delivery we usually print with Sticker Mule and it has been very reliable and quick (same week delivery).

**Comment by pocmo** [comment]
Another idea (that is quick to hack together) is to create a random insider-joke sticker with the emojis from our milestones :)

**Comment by Amejia481** [comment]
Maybe android of one side of the box and AC in the other side
![img_20181113_104344](https://user-images.githubusercontent.com/773158/48425105-203c3d80-e732-11e8-9b27-9038b82b46cf.jpg)

**Comment by jrconlin** [comment]
One other lesson learned from doing my own stickers: 
When sending the art in to print, you want to use "printer palate" colors rather than screen colors. (IOW: make sure the colors are brighter than you probably want.) If you've got access to an inkjet or color laser, Do a test print! and refine the colors appropriately. 

(Mine came out a bit darker than I would prefer. They're fine, because I wanted an "art deco" look to them, but I'd have preferred a bit brighter.)

**Comment by jonalmeida** [comment]
@jrconlin or @aminalhazwani , how detailed can the image be? We wanted to add emojis on to the crate but I'm not sure if they'll get printed out clearly.

Recommended image resolution/size?

**Comment by jrconlin** [comment]
I'm really impressed by the print quality. I gave them an SVG, and from what I can tell, it's printed at at least 600dpi. 
my 2x3" sticker: https://photos.app.goo.gl/Sek5PSMFpmvJrbkG8
detail on the ship name: https://photos.app.goo.gl/xk5FTMCba8uLxYPz8

**Comment by jonalmeida** [comment]
I took @jrconlin 's idea of a crate and slapped on a few things:
 - Android logo (for obvious reasons)
 - Mozilla dino (represents Gecko since we don't have a logo for that)
 - Servo (because we support it too)
 - The emojis are what we use for releases and also inside jokes.

I also made two simple options for a "reference" browser. `*&` is more of a "de-reference" though.

Thoughts? We'd like to put these into print this week.

![crate-2](https://user-images.githubusercontent.com/1370580/48683765-a31f2700-eb7c-11e8-8926-634abdf1930c.png)
![reference-001](https://user-images.githubusercontent.com/1370580/48683763-a31f2700-eb7c-11e8-8423-8e363124acb7.png)
![reference-002](https://user-images.githubusercontent.com/1370580/48683764-a31f2700-eb7c-11e8-87a4-c73858011e53.png)
